src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -0,0 +1,127 @@
+(ns status-im.contexts.wallet.create-account.new-keypair.check-your-backup.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im.contexts.wallet.common.temp :as temp]
+    [status-im.contexts.wallet.create-account.new-keypair.check-your-backup.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def secret-words-count 12)
+
+(defn- random-selection
+  []
+  (->> (range secret-words-count)
+       (shuffle)
+       (take 4)
```

Comment:
4 is the columns-length?
perhaps we put it in a var, also same for rows-length of 3 below?

Reply:
4 in the number of questions the user will be asked (4 question items). I can put it in a var